src/Badge.js

```diff
@@ -57,7 +60,7 @@ const Badge = styled.div`
   font-weight: 600;
   text-transform: uppercase;
   letter-spacing: ${themeGet('letterSpacings.caps')};
-  ${space} ${type};
+  ${space} ${type} ${color};
```

Comment:
NB: we can probably rethink whether or not this component extends the `Box` component in v2, which would mean all the Box style props would work on this component as well